projects/standard-rulesets/src/breaking-changes/index.ts

```diff
@@ -67,6 +75,10 @@ const breakingChangesRules = [
   preventResponsePropertyRemoval,
   preventResponsePropertyTypeChange,
   preventResponseStatusCodeRemoval,
+  preventRequestPropertyRangeChanges,
+  preventResponsePropertyRangeChanges,
+  preventRequestArrayLengthChanges,
+  preventResponseArrayLengthChanges,
```

Comment:
How should we handle new rules. What you've got here will start failing on things that weren't failing before. 

We have two options:
- We expand the breaking change definition as part of upgrading, and you can turn off rules as you upgrade
- We can "freeze" the current set of breaking change rules, and you can opt into extra breaking change rules. When we "major version" we can turn on all breaking changes.

Reply:
That's worth a longer convo on. These are change rules, so nothing that was passing will fail. But subsequent classes of changes you make might start failing